html_css/v2/foundations/html-foundations/links-and-images.md

```diff
@@ -0,0 +1,209 @@
+### Introduction
+
+Links are one of the key features of HTML. They allow us to link to other html pages on the web. In fact, this is why it's called the web. The internet is essentially a web of html pages connected to one another with links.
+
+In this lesson, we will learn how to create links and add some visual flair to our websites by embedding images.
+
+### Learning Outcomes
+
+- How to create links to pages on other websites on the internet
+- How to create links to other pages on your own websites
+- The difference between absolute and relative links
+- How to create images in HTML
+
+### Preparation
+
+To get some practice using links and images throughout this lesson we need an html project to work with. Create a new folder named `odin-links-and-images`.
+
+Within that folder create a new file named `index.html` and paste the following into it:
+
+~~~html
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Odin Links and Images</title>
+    <meta charset="UTF-8">
+  </head>
+
+  <body>
+    <h1>Homepage</h1>
+  </body>
+</html>
+~~~
```

Comment:
Instead of having students copy+paste the boilerplate from the lesson, would it be good to give them more practice with it by having them write the boilerplate out themselves and telling them what to use as contents, e.g. "Write out the boilerplate, and give the title element X for contents and add this element to the body"?

Reply:
There's already a lesson that covers the boilerplate.... and honestly do _any_ of us actually type this stuff out manually at this point?